delfi_attributes.json

```diff
@@ -0,0 +1,1022 @@
+	  "CONDITION": {
+		"osm_tags": {
+		  "tourism": "information",
+		  "information": "map",
+		  "map_type": "public_transport",
+		  "map_size": "site"
+		}
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2021,
+	  "LABEL": "Breite der niveaugleichen Wege",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": [
+			"footway",
+			"path"
+		  ]
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"width": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2030,
+	  "LABEL": "Tür vorhanden",
+	  "CONDITION": {
+		"osm_tags": {
+		  "entrance": true
+		},
+		"osm_element": "Node"
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2032,
+	  "LABEL": "Art der Tür/des Zugangs",
+	  "CONDITION": {
+		"osm_tags": {
+		  "entrance": true
+		},
+		"osm_element": "Node"
+	  },
+	  "TAGS": {
+		"door": [
+		  "hinged",
+		  "sliding",
+		  "revolving",
+		  "swinging",
+		  "no"
+		]
+	  }
+	},
+	{
+	  "DELFI_ID": 2033,
+	  "LABEL": "Art der Türöffnung",
+	  "CONDITION": {
+		"osm_tags": {
+		  "entrance": true
+		},
+		"osm_element": "Node"
+	  },
+	  "TAGS": {
+		"automatic_door": [
+		  "yes",
+		  "no",
+		  "button",
+		  "motion"
+		]
+	  }
+	},
+	{
+	  "DELFI_ID": 2034,
+	  "LABEL": "Lichte Breite der Türöffnung/des Zugangs",
+	  "CONDITION": {
+		"osm_tags": {
+		  "entrance": true
+		},
+		"osm_element": "Node"
+	  },
+	  "TAGS": {
+		"width": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2040,
+	  "LABEL": "Gleisüberquerung erforderlich",
+	  "CONDITION": [
+		{
+		  "osm_tags": {
+			"railway": "crossing"
+		  },
+		  "osm_element": "Node"
+		},
+		{
+		  "osm_tags": {
+			"railway": "tram_crossing"
+		  },
+		  "osm_element": "Node"
+		}
+	  ],
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2050,
+	  "LABEL": "Unbefestigter Bodenbelag",
+	  "CONDITION": [
+		{
+		  "osm_tags": {
+			"highway": [
+			  "footway",
+			  "path"
+			]
+		  },
+		  "osm_element": "OpenWay"
+		},
+		{
+		  "osm_tags": {
+			"public_transport": "platform"
+		  }
+		}
+	  ],
+	  "TAGS": {
+		"surface": [
+		  "paved",
+		  "unpaved",
+		  "asphalt",
+		  "concrete",
+		  "paving_stones",
+		  "sett",
+		  "cobblestone",
+		  "metal",
+		  "wood",
+		  "compacted",
+		  "fine_gravel",
+		  "pebblestone",
+		  "earth",
+		  "grass",
+		  "ground",
+		  "gravel",
+		  "mud",
+		  "sand",
+		  "snow",
+		  "ice",
+		  "salt",
+		  "woodchips",
+		  "grass_paver",
+		  "gravel_turf",
+		  "unhewn_cobblestone",
+		  "shell",
+		  "grasscrete",
+		  "rock",
+		  "clay",
+		  "debris",
+		  "dirt",
+		  "grass_paving",
+		  "plastic",
+		  "tar_and_chip",
+		  "woodchips"
+		]
+	  }
+	},
+	{
+	  "DELFI_ID": 2070,
+	  "LABEL": "Bodenindikatoren vorhanden",
+	  "CONDITION": [
+		{
+		  "osm_tags": {
+			"highway": [
+			  "footway",
+			  "path",
+			  "steps"
+			]
+		  }
+		},
+		{
+		  "osm_tags": {
+			"railway": "platform"
+		  }
+		},
+		{
+		  "osm_tags": {
+			"public_transport": "platform"
+		  }
+		}
+	  ],
+	  "TAGS": {
+		"tactile_paving": [
+		  "yes",
+		  "no",
+		  "contrasted"
+		]
+	  }
+	},
```

Comment:
I got some new information here. So `highway= footway/path` should actually be split of into the DELFI attribute 2072.